### ISSUE TITLE
No sound in output file

### ISSUE THREAD

**Issue by VKdeveloper** [MasayukiSuda/Mp4Composer-android#80]
After we implement the filter, provide input file path and use Mp4Composer class, the output file has no audio.
Please check into this issue as soon as possible.

Environment:
Android : Android 10
Device: Mi A3

**Comment by MasayukiSuda** [comment]
Please write your sample code.

**Comment by remorr25** [comment]
No audio in output file below the code I have integrated in my code

                 new GPUMp4Composer(srcMp4Path, destMp4Path)
                .filter(new GlFilterGroup(FilterType.createGlFilter(filterTypes.get(select_postion), getApplicationContext())))
                .listener(new GPUMp4Composer.Listener() {
                    @Override
                    public void onProgress(double progress) {
                     }

                    @Override
                    public void onCompleted() {
                    }

                    @Override
                    public void onCanceled() {
                        Log.d("resp", "onCanceled");
                    }

                    @Override
                    public void onFailed(Exception exception) {
                        Log.d("resp",exception.toString());
                    }
                })
                .start();

**Comment by MasayukiSuda** [comment]
`GPUMp4Composer` has nothing to do with this project.

**Comment by VKdeveloper** [comment]
Following is the code:
```
Bitmap myLogo = //some bitmap
         Bitmap bitmap_resize=Bitmap.createScaledBitmap(myLogo, 100, 100, false);
         GlWatermarkFilter filter=new GlWatermarkFilter(bitmap_resize, GlWatermarkFilter.Position.LEFT_TOP);
         new Mp4Composer(Variables.app_folder+item.video_id+"default"+".mp4",
                Variables.app_folder+child.id+".mp4")
                 .mute(false)
                 .filter(filter)
                .listener(new Mp4Composer.Listener() {
                    @Override
                    public void onProgress(double progress) {
                        Log.d("progess",""+(int) (progress*100));
                    }

                    @Override
                    public void onCompleted() {
                          Log.i("LOG","Prcoessing completed");
                    }

                    @Override
                    public void onCanceled() {
                        Log.d("resp", "onCanceled");
                    }

                    @Override
                    public void onFailed(Exception exception) {
                        Log.d("resp",exception.toString());
                    }
                })
                .start();
```
Above code mergers bitmap perfectly at left top but video is saved without audio. Please check.

**Comment by VKdeveloper** [comment]
> 
> 
> Please write your sample code.

Please check above code.

**Comment by MasayukiSuda** [comment]
Does it work on other devices?

**Comment by VKdeveloper** [comment]
> 
> 
> Does it work on other devices?

Yes it works on my old device of Micromax Canvas Spark (India) (2016) , which has following specs:
- Android 5.1.1
- MT6582 Processor
- 1 GB RAM

I have tested on Redmi note 3S and it fails. (Android 6.0, 3GB RAM)
Tester has also tested on a Samsung device and it fails there too. (Android 7.0, 3GB RAM)

Will update Samsung device model after some time.

It is my very kind request, to find the issue. I am stuck in very crucial moment right now. 
Cannot deliver the software to the client without audio in the video.

May be can you just figure out some workaround to patch the bug and release the update later ?

**Comment by evilrat** [comment]
same issue here did you got any fix

**Comment by VKdeveloper** [comment]
@MasayukiSuda Have you found any solution ?

**Comment by Krish140994** [comment]
Same issue , Anyone found any solution

**Comment by persadaditya** [comment]
maybe you tested in release with proguard enabled.
try with debug mode or disabled proguard.
.
I had tested another library with same issue like this
looking forward to check your test again
